gst/tensor_decoder/tensordec.c

```diff
@@ -468,6 +509,13 @@ gst_tensordec_set_property (GObject * object, guint prop_id,
     case PROP_MODE:
       self->mode = g_value_dup_string (value);
       break;
+    case PROP_MODE_OPTION1:
+      if (g_strcmp0 (self->mode, "image_labeling") == 0) {
+        self->tensordec_mode_image_label.image_labeling_info.label_path =
+            g_value_dup_string (value);
```

Comment:
IMHO, since g_value_dup_string( ) newly allocates memory to copy the value, self->tensordec_mode_image_label.image_labeling_info.label_path should be freed after it is used.

Reply:
I will add tensor_dec_stop function for free self->mode and self>tensordec_mode_image_label.image_labeling_info.label_path